tracer/test/Datadog.Trace.Security.Unit.Tests/IAST/Tainted/evidence-redaction-suite.yml

```diff
@@ -6,6 +6,7 @@ suite:
       $1:
         - access_key_id
```

Comment:
This file has been updated with the latest changes in https://github.com/DataDog/experimental/blob/main/teams/asm/vulnerability_schema/vulnerability_schema.json